packages/sdks/snippets/gen1-remix/app/routes/editable-region.tsx

```diff
@@ -0,0 +1,49 @@
+// routes/advanced-child.tsx
+import { Builder, BuilderComponent, builder } from "@builder.io/react";
+import type { LoaderFunctionArgs } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { CustomColumns } from "./components/CustomColumns";
+
+builder.init("ee9f13b4981e489a9a1209887695ef2b");
+
+Builder.registerComponent(CustomColumns, {
+  name: "MyColumns",
+  inputs: [
+    {
+      name: "column1",
+      type: "uiBlocks",
+      broadcast: true,
+      hideFromUI: true,
+      defaultValue: {
+        blocks: [],
+      },
```

Comment:
do we need the `defaultValue`s here at all? I see that the other component's `tabList` has `defaultValue: []` which is different.

ideally we can completely remove the default value to make code simpler. but if its needed, lets use the more brief version.

Reply:
@samijaber I think defaultValue here serves as a placeholder for displaying the "**+Add Blocks**" element on the UI. Without this, we could not render it on the screen.

Reply:
@samijaber I will give it a try and see if i can remove and make it work